packages/snjs/lib/services/features_service.ts

```diff
@@ -87,6 +91,7 @@ export class SNFeaturesService extends PureService<void> {
           const userKey = userKeyMatch[0];
           await this.settingsService
             .updateSetting(SettingName.ExtensionKey, userKey, true);
+          await this.itemManager.setItemToBeDeleted(item.uuid);
```

Comment:
@mobitar wait I've just realized, don't we need the extension repo items for our current extension manager logic?

Reply:
We do. We can either coordinate the release of this with the removal of ext-manager, or we can modify the Repo object to say `content.migratedToUserSetting = true` and sync it.